templates/index.html

```diff
@@ -57,8 +57,14 @@ <h3 class="fs-title">
 </div>
 
 {% if CASS_HOME_EXTRAS %}
-    {% set home_extras = pages[7] %}  <!-- Set to 6 to grab homepage extras --> 
+    {% set home_extras = pages[7] %}  <!-- Set to 7 to grab homepage extras --> 
     {{ home_extras.content }}
 {% endif %}
 
+{% if OSL_HOME_EXTRAS %}
+    {% set home_extras = pages[4] %}  <!-- Set to 4 to grab homepage extras --> 
```

Comment:
you should be able to select specifically about.rst, something like 
`for page in pages if page.title == About`
or, use a 'front-page' tag and display everything with that tag, would allow future additions to the front page, but would require content changes....